### PR TITLE
Add JBoss editorial for 2022-12-01

### PR DESCRIPTION
**Pull request by rpelisse**

### CODE REVIEW

File: src/content/posts/weekly-2022-12-01.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Last, but not the least, two more, deep articles about Quarkus : one on link:https://quarkus.io/blog/reactive-crud-performance-case-study/[Reactive CRUD performance case study] and the other on link:https://quarkus.io/blog/redis-job-queue-reloaded/[Redis Job Queue - Reloaded] (a followup on Clément's previous article on link:https://quarkus.io/blog/redis-job-queue/[How to implement a job queue with Redis].
+
+== A good time to learn new (or old) things
+
+The JBoss communities are always producing articles and deep dive about their technology, but the last two weeks especially have seen the release of some introductions and presentations ones. The first one is one of my favorites, because it talks about an unsung hero of Wildfly ecosystem: link:https://jberet.github.io/jberet-intro/[Introducing The Java Batch Processing API And JBeret Implementation]. If you have no idea what this is about, please go check it out, this is bound to make your life easier!
```
Comment (LightGuard) [inline src/content/posts/weekly-2022-12-01.adoc:18]: This first sentence doesn't sound right, I'm not sure what you're saying.

File: src/content/posts/weekly-2022-12-01.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+After this is definitely this humble article entitled link:http://www.mastertheboss.com/java-ee/jakarta-ee/a-maven-starter-for-jakarta-ee-projects/[A Maven starter for Jakarta EE projects]. Indeed, Maven has been at the heart of Java (and thus JBoss) technology for a long while now (around two decades), so such an introduction, for newcomers like experiences developers alike, is certainly nice to see.
```
Comment (LightGuard) [inline src/content/posts/weekly-2022-12-01.adoc:20]: newcomers and experienced developers alike

File: src/content/posts/weekly-2022-12-01.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+== Techbytes
+
+If you need some more involved than a quickstarter, there are two articles coming from the Kogito and Drools ecosystem: link:https://blog.kie.org/2022/11/kogito-serverless-workflow-event-formats.html[Kogito serverless workflow event formats] and link:https://blog.kie.org/2022/11/drools-reactive-messaging-processing.html[Drools reactive messaging processing].
```
Comment (LightGuard) [inline src/content/posts/weekly-2022-12-01.adoc:24]: something more

File: src/content/posts/weekly-2022-12-01.adoc [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Last, but not the least, two more, deep articles about Quarkus : one on link:https://quarkus.io/blog/reactive-crud-performance-case-study/[Reactive CRUD performance case study] and the other on link:https://quarkus.io/blog/redis-job-queue-reloaded/[Redis Job Queue - Reloaded] (a followup on Clément's previous article on link:https://quarkus.io/blog/redis-job-queue/[How to implement a job queue with Redis].
+
+== Ansible support for JBoss runtimes
+
+It's not strictly speaking JBoss community, but I'm too proud of this baby of mine to keep quiet about it. Since over a year, I've been part of an initiative to provide better integration between the automation tool Ansible and many of the middleware solutions being developed by the JBoss Community. We are quite to have already Ansible collection (extension for Ansible) that ease the setup of link:https://github.com/ansible-middleware/wildfly[Wildfly], link:https://github.com/ansible-middleware/infinispan[Infinispan], link:https://github.com/ansible-middleware/keycloak[Keycloak], and link:https://github.com/ansible-middleware/amq[AMQ Broker].
```
Comment (LightGuard) [inline src/content/posts/weekly-2022-12-01.adoc:30]: quite happy to already have ?

File: src/content/posts/weekly-2022-12-01.adoc [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+If you are curious about the potential of those collections and how Ansible can help when you deploy JBoss runtimes, please check this (awesome) demo on link:https://events.experiences.redhat.com/widget/redhat/rhaf22/SessionCatalog2022/session/16579300056760019EPY[Ansible Fest Session Catalog: Managing your Red Hat Middleware estate from the edge to the cloud with Ansible].
+
+== Releases, releases, releases...
+
+Releasing is crucial for Open Source project and releasing often is certainly the mark of healthy one, so it comes with no surprises that the last two weeks have already seen their fair share of releases:
+
+* link:https://quarkus.io/blog/quarkus-2-14-2-final-released/[Quarkus 2.13.5.Final and 2.14.2.Final] including a fix for link:https://access.redhat.com/security/cve/cve-2022-4116[CVE-2022-4116].
+* link:https://quarkus.io/blog/quarkus-3-0-0-alpha1-released/[Quarkus 3.0.0.Alpha1] a first iteration of the Jakarta 10, check it out!
```
Comment (LightGuard) [inline src/content/posts/weekly-2022-12-01.adoc:41]: Quarkus 3 has Jakarta EE 10 stuff?

**Review by LightGuard — CHANGES_REQUESTED** [review]

File: src/content/posts/weekly-2022-12-01.adoc (hunk 5 above)
Comment (rpelisse) [reply src/content/posts/weekly-2022-12-01.adoc:41]: @LightGuard https://quarkus.io/blog/quarkus-3-0-0-alpha1-released/

Comment (LightGuard) [reply src/content/posts/weekly-2022-12-01.adoc:41]: Huh. Okay, cool!

**Comment by LightGuard** [comment]
Looks good

**Review by LightGuard — APPROVED** [review]

**Pull request merged**